pages/common/flutter-pub.md

```diff
@@ -0,0 +1,21 @@
+# flutter pub
+
+> Flutter's own package manager.
+> Packages are available on <https://pub.dev>.
+> More information: <https://docs.flutter.dev/packages-and-plugins/using-packages>.
+
+- Download / Update all packages specified in `pubspec.yaml`:
+
+`flutter pub get`
+
+- Add a package dependency to an app:
+
+`flutter pub add {{package1, package2 ...}}`
```

Comment:
```suggestion
`flutter pub add {{package}}`
```

Reply:
> I personally find it useful to know it is possible to add several packages in one command

Oh, I am not a Flutter user myself and all the tutorials I can find were just passing a single package. So it is indeed useful to know that several packages can be added in the command. 

Is the `,` between packages necessary?